about.html

```diff
@@ -0,0 +1,91 @@
+  
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/bdd8845187.js" crossorigin="anonymous"></script>
+  
+  <title>About MoP.</title>
+</head>
+<body>
+<main>
+  <header>
+    <nav class="navigation d-flex align-items-center justify-content-between">
+      <i class="fa-solid fa-bars open-menu"></i>
+      <a href="./index.html">MoP.™</a>
+      <ul class="navigation__list">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="index.html#speakers">Speakers</a></li>
+      </ul>
+    </nav>
+  </header>
+  <ul class="navigation__mobile">
+    <li><a href=""><img id="close-menu" class="close-menu" src="./images/close.png" alt="close"></a></li>
+     <li><a href="./index.html" class="navigation__mobile__links externalLink">Home</a></li>
+     <li><a href="index.html#program" class="navigation__mobile__links externalLink">Program</a></li>
+     <li><a href="index.html#speakers" class="navigation__mobile__links externalLink">Speakers</a></li>
+  </ul>
+  <section class="about">
+    <p class="about__para1 text-center">
+      "Greetings, students of Philosophy"
+    </p>
+    <h1>
+      Masters of Philosophy <br>
+      ANNUAL CONGRESS OF DISCOURSE <br>
+      2022 <br>
+    </h1>
+    <p class="about__para2 p-3">
+      The Congress brings together the most brilliant minds in modern Philosophy
+      to share knowledge and discuss about the latest advancements in the numerous 
+      fields of philosophy. We have philosphy professors from all over the globe 
+      who grace us with their attendance to give passionate lectures on the translations
+      of all our favorite ancient philosophers. 
+    </p>
+  </section>
+
+  <section class="statement d-flex flex-column">
+    <div class="statement__container">
+      <h2>
+        Mission
+      </h2>
+      <p>
+        To keep the fire of philosophy burning and bring the greatest minds
+        together in order to further the study of all matters concerning
+        the enlightment of man.
+      </p>
+    </div>
+    <div class="statement__container">
+      <h2>
+        Vision
+      </h2>
+      <p>
+        To use our new-found hologram technology to reach more scholars and 
+        provide live lectures of all the ancient philosophers in more than 50 languages
+        all thanks to the efforts of our translation teams.
+      </p>
+    </div>
+  </section>
+  <section class="enquiries p-3">
+    <h2 class="text-center">
+      For any enquiries about the Congress of Discourse, 
+      feel free to contact us at:
+    </h2>
+    <a href="#">mastersofphilosophy@gmail.com</a>
+  </section>
+  <footer class="copyright">
+    <p>
+      MoP © 2022
+    </p>
+  </footer>
```

Comment:
- [ ] According to the design guidelines, your footer should have a white background on mobile devices. Please implement this change.
### Your Design
![image](https://user-images.githubusercontent.com/46839548/170464536-c867602d-5005-48c6-bfb7-7fd3313565ad.png)
### Design Guideline
![image](https://user-images.githubusercontent.com/46839548/170464592-44dc75e6-9a81-4cf3-9d0a-4aa59371d392.png)

- [ ] The footer on the homepage should also have a white background according to the design guideline.


- [ ] I noticed that You have omitted a section before the footer which shows the partners and their logos. Please include it 🙏 
### Guideline
![image](https://user-images.githubusercontent.com/46839548/170476709-ce3807f4-fed0-47b9-8bda-be39a0cd10bb.png)